### ISSUE TITLE
Radarr using a lot of CPU constantly (Synology NAS)

### ISSUE THREAD

**Issue by BladesKnight** [Radarr/Radarr#1718]
Installed the latest available version (0.2.0.778) and now Radarr is constantly using about 25% of CPU. Activity monitor on Synology just says that its "Main" using up the CPU resource.

Check first that your problem is not listed in our wiki section:
* https://github.com/Radarr/Radarr/wiki/Common-Problems
* https://github.com/Radarr/Radarr/wiki/FAQ

Provide a description of the feature request or bug here, the more details the better. 
Please also try to include the following if you are reporting a bug

Radarr Version: 0.2.0.778

**Logs:**

Please use the search bar and make sure you are not submitting an already submitted issue.
Visit our [Discord server](https://discord.gg/NWYch8M) for support or longer discussions.

**Comment by galli-leo** [comment]
What version of mono are you using?

**Comment by BladesKnight** [comment]
I have 2 instances of Mono running and I don't know which one it's using..
Synocommunity's 4.6.2.7-8 and Xamarin's 4.6.2-0095.

**Comment by galli-leo** [comment]
@BladesKnight Can you try upgrading your mono version to 4.8 or 5.0? Someone else saw a big decrease in CPU usage.

**Comment by BladesKnight** [comment]
There are no synology packages for version 4.8 or 5.0. What I have is the latest available.

**Comment by wvkreg** [comment]
I use Mono 5.0 with Radarr in Docker container version Radarr Ver. 0.2.0.778
Still CPU usage spikes sometimes but is not constantly high.

**Comment by BladesKnight** [comment]
Have upgraded to Mono 5.0.1.1 and running Radarr in Docker and CPU is still relatively high hovering at 20-25%. 

Noticed the same in Sonarr but they seem to have fixed it in their latest version 2.0.0.4949 with a "Fixed : Regression in http redirect logic causing failed grabs and >25% cpu usage."

Maybe the same thing is affecting Radarr?

**Comment by galli-leo** [comment]
I will try to pull that change into Radarr

**Comment by JoeB99** [comment]
Did this change make it into Radarr? I'm seeing mono using 100% CPU constantly..

**Comment by JoeB99** [comment]
Has anything canged on this? Still 100% CPU usage. Can't run radarr at all as mono wants the whole CPU!

**Comment by Ritte88** [comment]
Comfirming this is still a problem, luckily me I run a dual-xeon server but I rather still it fixed.

**Comment by jj-uk** [comment]
Appears to have been fixed for me on FreeNAS in version 0.2.0.910. Thank you.

**Comment by jaimevisser** [comment]
The current linuxserver.io docker image reproduces this behaviour nicely. Running it on ubuntu 17.10 with latest docker-ce.
![screen shot 2018-01-24 at 09 13 09](https://user-images.githubusercontent.com/3132806/35321199-d2c23852-00e6-11e8-8ff6-5dfc5a7ab74d.png)

**Comment by fraenkDUS** [comment]
Same problem here with linuxserver.io docker/latest on Synology and Radarr 0.2.0.980

**Comment by galli-leo** [comment]
Finally got around to pull in those changes from Sonarr, maybe it will help (new release in a few hours).

**Comment by Nightmarel33t** [comment]
has there been any update on the high CPU usage my mono sits there constantly at 30%

![image](https://user-images.githubusercontent.com/20948070/37428799-4d65478e-27c5-11e8-84b2-88285cac52cf.png)

**Comment by Nightmarel33t** [comment]
thats on version 0.2.0.995 -

**Comment by galli-leo** [comment]
@Nightmarel33t What system are you running and was Radarr actively doing something, e.g. RSS Sync updating info, etc.?

**Comment by Taymurf** [comment]
I am having the same issue on my Synology DS918+ with radarr in docker, I'm using the Linuxserver image and I have 2 processes of mono running for about 20-30% CPU each at all times. My idle CPU usage right now is 60%.

**Comment by Taymurf** [comment]
I believe that one of the mono processes is from sonarr and one is from radarr (both programs running in docker) but when I stop sonarr the CPU stays up and when I stop radarr they both go down. I do also have dockers for deluge, jackett and hydra but I'm not sure if those have any effect since the docker overview shows minimal usage by them. [Here](https://i.imgur.com/kY0TuZ7.png) is a screenshot of my config file for radarr including sizes.

I remade the radarr container and it does seem to have helped quite a bit (at least for the last 10 minutes) so I'm wondering if something got stuck in a loop in the container that got fixed when I remade it? I used the same image in both containers.
